README.md

```diff
@@ -136,12 +136,12 @@ ErrorResponse = {
 When client uses
 commands [`session.subscribe`](https://w3c.github.io/webdriver-bidi/#command-session-subscribe)
 and [`session.unsubscribe`](https://w3c.github.io/webdriver-bidi/#command-session-unsubscribe)
-with `channel`, the subscriptions are handled per channel, and the corresponding
-`channel` filed is added to the event message:
+with `goog:channel`, the subscriptions are handled per channel, and the corresponding
+`goog:channel` filed is added to the event message:
 
 ```cddl
 Event = {
-  channel?: text,
+  goog:channel?: text,
```

Comment:
I think this might be invalid CDDL syntax

Reply:
good point. I believe I fixed it now